specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/examples/WafListPolicies.json

```diff
@@ -43,8 +43,7 @@
                     {
                       "state": "Enabled",
                       "matchVariable": "RequestIPAddress",
-                      "selectorMatchOperator": "EqualsAny",
```

Comment:
nit: Both the lines are showing up on the diff as modified while only 1 line has been modified.